apps/simplestpp/metadata.json

```diff
@@ -1,6 +1,6 @@
 {
   "id": "simplestpp",
-  "name": "Simplest++ Clock",
```

Comment:
I think it might be better to use `shortName` here (which appears on the Bangle) - I'll do that now though.